### ISSUE TITLE
Tolgee Tools for Firefox

### ISSUE THREAD

**Issue by KaKi87** [tolgee/tolgee-platform#2297]
Hi,

It would be nice to have the Tolgee Tools extension on Firefox.

Thanks

**Comment by stepan662** [comment]
I agree, this would be a nice thing for contribution. I'm not sure how different are extensions in Firefox, but I guess the main logic should be the same as in https://github.com/tolgee/chrome-plugin.

**Comment by stepan662** [comment]
https://github.com/tolgee/chrome-plugin/pull/31
